website/src/pages/showcase.js

```diff
@@ -59,56 +62,140 @@ const renderLinks = app => {
-      <div className="showcaseSection">
-        <div className="prose">
-          <h1>Who's using React Native?</h1>
+      <Section background="dark">
+        <div className="sectionContainer headerContainer">
+          <h1>
+            Who is using <span>React Native</span>?
+          </h1>
           <p>
             Thousands of apps are using React Native, from established Fortune
-            500 companies to hot new startups. If you're curious to see what can
-            be accomplished with React Native, check out these apps!
+            500 companies to hot new startups.
+            <br />
+            If you are curious to see what can be accomplished with React
+            Native, check out these apps!
+          </p>
+        </div>
+      </Section>
+      <Section>
+        <div className="showcaseSection">
+          <h2 className="withLogo">
+            <ThemedImage
+              alt="Meta logo"
+              width={140}
+              sources={{
+                light: useBaseUrl('/img/meta_positive_primary.svg'),
+                dark: useBaseUrl('/img/meta_negative_primary.svg'),
+              }}
+            />
+          </h2>
+          <p className="showcaseSectionDescription">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam sit
+            amet ex ullamcorper, dignissim elit ac, ornare elit. Donec iaculis,
+            erat in pharetra euismod, sem ipsum varius urna, nec hendrerit nisi
+            urna sed lorem.
           </p>
+          <div className="logos">{meta.map(renderApp)}</div>
         </div>
-        <div className="logos">{apps.map(renderApp)}</div>
-        <a
-          className="form-button"
-          href="https://forms.gle/BdNf3v5hemV9D5c86"
-          target="_blank">
-          Fill out this form to apply to the customer spotlight.
-        </a>
-        <p>
-          A curated list of{' '}
-          <span>
+        <div className="showcaseSection">
+          <h2 className="withLogo">
+            <ThemedImage
+              alt="Microsoft logo"
+              width={180}
+              sources={{
+                light: useBaseUrl('/img/microsoft-logo-gray.png'),
+                dark: useBaseUrl('/img/microsoft-logo-white.png'),
+              }}
+            />
+          </h2>
+          <p className="showcaseSectionDescription">
+            At Microsoft, we leverage the power of React Native to enable our
+            developers to deliver excellent customer experiences in some of our
+            most well known apps.
+            <br />
+            We don't stop at mobile platforms either, we leverage it to target
+            desktop too - find out more in our{' '}
+            <a
+              href="https://microsoft.github.io/react-native-windows/resources-showcase"
+              target="_blank">
+              dedicated showcase
+            </a>{' '}
+            for React Native Windows and macOS.
```

Comment:
```suggestion
            Microsoft leverages the power of React Native to deliver excellent customer experiences in some of its most well known apps.
            <br />
            Microsoft doesn't stop at mobile platforms either -- Microsoft leverages React Native to target desktop too! Find out more in the {' '}
            <a
              href="https://microsoft.github.io/react-native-windows/resources-showcase"
              target="_blank">
              dedicated showcase
            </a>{' '}
            for React Native Windows and macOS.
```
final copy reviewed with the PMs :)